docs/controls/MasterDetailsView.md

```diff
@@ -19,19 +19,31 @@ The [MasterDetailsView Control](https://docs.microsoft.com/dotnet/api/microsoft.
           NoSelectionContentTemplate="{StaticResource NoSelectionContentTemplate}"
           CompactModeThresholdWidth="720">
 </controls:MasterDetailsView>
-```
+```
 
 ## Sample Output
 
 ![MasterDetailsView animation](../resources/images/Controls/MasterDetailsView.gif)
 
+## BackButtonBehavior
+The MasterDetailsView control uses the first Frame in the parent tree to handle back navigations - moving from details to master in compact mode. For example, if the Details view state is active when attempting back navigation on the Frame will move the MasterDetailsView view to the Master view and cancel the back navigation.
+
+The MasterDetailsView will handle back button visibility of the SystemNavigationManager back button, a parent NavigationView back button, or an inline back button. Use the `BackButtonehavior` property to control the behaviour:
+- `Automatic` to let the control decide which back button to make visible/enabled, either the system back button or a back button in a parent NavigationView. 

```

Comment:
Auto will show inline as well.

- if the system back button is not already visible
- If there is no navview OR, if there is a nav view, but no Frame

Reply:
updated